modules/ROOT/pages/business-groups.adoc

```diff
@@ -1,171 +1,74 @@
-
-[[control-business-group-access]]
-=== View User and Team Access 
-
-You can view a list of users or teams that have access to a business group or environment and filter by their assigned permissions.
-
-include::partial$include-nav-business-groups.adoc[]
-
-. Select the *Access Overview* tab. 
-+
-A list of users in the current business group is shown.
- 
-.. Select the business group or environment from the dropdown to view a list of users with access to the selected business group or environment.
-.. Select a permission from the dropdown to see a list of users who have the selected permission in the selected business group or environment. 
-. To see teams with access to the selected business group or environment, select *Teams* from the dropdown.
-. To see teams that have specific permissions assigned, select the permission from the dropdown. 
-+
-Permissions that are granted directly to users outside of the Teams feature aren't shown. 
-
-=== View Roles Associated With the Business Group
-
-include::partial$include-nav-business-groups.adoc[]
-
-. Select the *Roles* tab. 
-+
-A list of roles in the current business group is shown. 
-
-[[delete-business-group]]
-== Delete a Business Group
-
-Only an organization administrator that belongs to a business group can delete it. The top-level (root) business group can't be deleted, even by an organization administrator.
-
-include::partial$include-nav-organizations.adoc[]
-
-. Click the *...* menu next to the business group to delete.
-. In the confirmation dialog, enter the name of the business group and then click *Delete*.
-
-[[redistribute-resources]]
-== Redistribute Resources Between Existing Business Groups
-
-You can redistribute resources (such as vCores, VPCs, load balancers, Small or Large Managed Flex Gateways, and so on) between business groups at the business group level only, not at the root organization level. 
-
-The business group hierarchy prevents a child business group from consuming additional resources from its parent group unless an organization administrator redistributes those resources to the child group. Resource distribution proceeds from root to parent to child. For example, you can't redistribute sandbox VCores to a child business group if the parent business group doesn't have any sandbox vCores available, even if the root organization has sandbox vCores available.
-
-Additionally, the root organization or parent business group can't reclaim resources allocated to a business group that isn't a direct child.
-
-To see resource utilization for the root organization, navigate to *Access Management > Subscription > Runtime Manage Subscription*.
+Users with Organization Administrator permission at any level automatically have that permission in any child business group created *within* that level. This permission inheritance is not retroactive; it doesn't apply to users who gain Organization Administrator permission *after* a child group is created.
```

Comment:
Might want to reconsider the bold for emphasis here, since we use bold for UI elements. Could be a note... It seems kind of important.